man/zram-generator.conf.md

```diff
@@ -0,0 +1,118 @@
+It is recommended to prefix all filenames in those subdirectories with a two-digit number and a dash, to simplify the ordering of the files.
+
+To disable a configuration file supplied by the vendor, the recommended way is to place a symlink to */dev/null* in the configuration directory in */etc/*,
+with the same filename as the vendor configuration file.
+
+## OPTIONS
+
+Each device is configured independently in its `[zramN]` section, where N is a nonnegative integer. Other sections are ignored.
+
+Devices with the final size of *0* will be discarded.
+
+* `host-memory-limit`=
+
+  Sets the upper limit on the total usable RAM (as defined by *MemTotal* in `/proc/meminfo`, confer proc(5)) above which the device will *not* be created.
+
+  This takes a nonnegative number, representing that limit in megabytes, or the literal string *none*, which can be used to override a limit set earlier.
+
+  Defaults to *none*.
+
+  For compatibility with earlier versions, `memory-limit` is allowed as an alias for this option.
+  Its use is discouraged, and administrators should migrate to `host-memory-limit`.
+
+* `zram-fraction`=
+
+  Defines the scaling factor of the zram device's size with relation to the total usable RAM.
+
+  This takes a nonnegative floating-point number representing that factor.
+
+  Defaults to *0.5*.
+
+* `max-zram-size`=
+
+  Sets the limit on the zram device's size obtained by `zram-fraction`.
+
+  This takes a nonnegative number, representing that limit in megabytes, or the literal string *none*, which can be used to override a limit set earlier.
+
+  Defaults to *4096*.
+
+* `compression-algorithm`=
+
+  Specifies the algorithm used to compress the zram device.
+
+  This takes a literal string, representing the algorithm to use.<br />
+  See */sys/block/zram0/comp_algorithm* for the list of algorithms supported by the currently running kernel.
+
+  If unset, none will be configured and the kernel's default will be used.
+
+## ENVIRONMENT VARIABLES
+
+Setting `ZRAM_GENERATOR_ROOT` during parsing will cause */proc/meminfo* to be read from *$ZRAM_GENERATOR_ROOT/proc/meminfo* instead,
+and *{/usr/lib,/usr/local/lib,/etc,/run}/systemd/zram-generator.conf* to be read from *$ZRAM_GENERATOR_ROOT/{/usr/lib,/usr/local/lib,/etc,/run}/systemd/zram-generator.conf*.
+
+## EXAMPLES
+
+The default configuration will yield the following:
+
+     zram device size [MB]
+         ^
+         │
+      4G>│               ooooooooooooo
+         │             o
+         │           o
+         │         o
+      2G>│       o
+         │     o
+         │   o
+    512M>│ o
+         0───────────────────────> total usable RAM [MB]
+           ^     ^       ^
+           1G    4G      8G
+
+## AUTHORS
+
+Written by Zbigniew Jędrzejewski-Szmek &lt;<zbyszek@in.waw.pl>&gt;,
+           Igor Raits &lt;<i.gnatenko.brain@gmail.com>&gt;,
+       and наб &lt;<nabijaczleweli@gmail.com>&gt;
```

Comment:
Hmm, maybe drop the AUTHORS section? Those tend to get out of date pretty quickly... And then there's always the issue where the cutoff point should be. We now have 7 authors by `git log`, and soon there'll be more. So I think it's OK to keep a short list in Cargo.toml, but I'd just stick to that.

Reply:
Fair enough; removed the list from both manpages and the note in Cargo.toml.